es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/ElasticSearchDAOV6.java

```diff
@@ -206,13 +195,13 @@ private void shutdownExecutorService(ExecutorService execService) {
         try {
             execService.shutdown();
             if (execService.awaitTermination(30, TimeUnit.SECONDS)) {
-                logger.debug("tasks completed, shutting down");
+                LOGGER.debug("tasks completed, shutting down");
             } else {
-                logger.warn("Forcing shutdown after waiting for 30 seconds");
+                LOGGER.warn("Forcing shutdown after waiting for 30 seconds");
                 execService.shutdownNow();
             }
         } catch (InterruptedException ie) {
-            logger.warn("Shutdown interrupted, invoking shutdownNow on scheduledThreadPoolExecutor for delay queue");
+            LOGGER.warn("Shutdown interrupted, invoking shutdownNow on scheduledThreadPoolExecutor for delay queue");
             execService.shutdownNow();
             Thread.currentThread().interrupt();
```

Comment:
Why do you explicitly call the current thread's interrupt, instead of letting the exception float up? At the very least you're loosing the context of the original exception.